src/realm-settings/DefaultGroupsTab.tsx

```diff
@@ -0,0 +1,228 @@
+
+  const [toggleRemoveDialog, RemoveDialog] = useConfirmDialog({
+    titleKey: t("removeConfirmTitle", { count: selectedRows.length }),
+    messageKey: t("removeConfirm", { count: selectedRows.length }),
+    continueButtonLabel: "common:delete",
+    continueButtonVariant: ButtonVariant.danger,
+    onConfirm: removeGroup,
+  });
+
+  if (!defaultGroups) {
+    return <KeycloakSpinner />;
+  }
+
+  return (
+    <>
+      <RemoveDialog />
+      {isGroupPickerOpen && (
+        <GroupPickerDialog
+          type="selectMany"
+          text={{
+            title: "realm-settings:addDefaultGroups",
+            ok: "common:add",
+          }}
+          onConfirm={(groups) => {
+            addGroups(groups);
+            toggleGroupPicker();
+          }}
+          onClose={toggleGroupPicker}
+        />
+      )}
+      {enabled && (
+        <Popover
+          bodyContent={
+            <Trans i18nKey="realm-settings-help:defaultGroups">
+              Default groups allow you...
+              <Link to={toUserFederation({ realm })} />.
+            </Trans>
+          }
+        >
+          <TextContent
+            className="keycloak__section_intro__help"
+            style={{
+              paddingLeft: "var(--pf-c-page__main-section--PaddingLeft)",
+            }}
+          >
+            <Text>
+              <QuestionCircleIcon /> {t("whatIsDefaultGroups")}
+            </Text>
+          </TextContent>
+        </Popover>
+      )}
+      <KeycloakDataTable
+        key={key}
+        canSelectAll
+        onSelect={(rows) => setSelectedRows([...rows])}
+        loader={loader}
+        ariaLabelKey="realm-settings:defaultGroups"
+        searchPlaceholderKey="realm-settings:searchForGroups"
+        toolbarItem={
+          <>
+            <ToolbarItem>
+              <Button
+                data-testid="openCreateGroupModal"
+                variant="primary"
+                onClick={toggleGroupPicker}
+              >
+                {t("addGroups")}
+              </Button>
+            </ToolbarItem>
+            <ToolbarItem>
+              <Dropdown
+                toggle={
+                  <KebabToggle
+                    onToggle={toggleKebab}
+                    isDisabled={selectedRows!.length === 0}
+                  />
+                }
+                isOpen={isKebabOpen}
+                isPlain
+                dropdownItems={[
+                  <DropdownItem
+                    key="action"
+                    component="button"
+                    onClick={() => {
+                      toggleRemoveDialog();
+                      toggleKebab();
+                    }}
+                  >
+                    {t("common:remove")}
+                  </DropdownItem>,
+                ]}
+              />
+            </ToolbarItem>
+          </>
+        }
+        actions={[
+          {
+            title: t("common:remove"),
+            onRowClick: async (group: GroupRepresentation) => {
```

Comment:
Nothing `async` is happening here.
```suggestion
            onRowClick: (group: GroupRepresentation) => {
```